lib/helper/Polly.js

```diff
@@ -0,0 +1,135 @@
+const Helper = require('../helper');
+const requireg = require('requireg');
+const { appendBaseUrl } = require('../utils');
+
+let PollyJS;
+
+class Polly extends Helper {
+  constructor(config) {
+    super(config);
+    this._setConfig(config);
+    PollyJS = requireg('@pollyjs/core').Polly;
+  }
+
+  static _checkRequirements() {
+    try {
+      requireg('@pollyjs/core');
+    } catch (e) {
+      return ['@pollyjs/core@^2.5.0'];
+    }
+  }
+
+  /**
+   * Start mocking network requests/responses
+   *
+   * ```js
+   * I.startMocking('Backend APIs');
+   * ```
+   */
+  async startMocking(title = 'Test') {
+    if (!this.helpers && !this.helpers.Puppeteer) {
+      throw new Error('Puppeteer is the only supported helper right now');
+    }
+    return this._connectPuppeteer(title);
+  }
+
+  /**
+   * Connect Puppeteer helper to mock future requests.
+   */
+  async _connectPuppeteer(title) {
+    const adapter = require('@pollyjs/adapter-puppeteer');
+
+    PollyJS.register(adapter);
+    const { page } = this.helpers.Puppeteer;
+    await page.setRequestInterception(true);
+
+    this.polly = new PollyJS(title, {
+      adapters: ['puppeteer'],
+      adapterOptions: {
+        puppeteer: { page },
+      },
+    });
+  }
+
+  /**
+   * Mock response status
+   *
+   * ```js
+   * I.mock('GET', '/api/users', 200);
+   * I.mock('ANY', '/secretsRoutes/*', 403);
+   * I.mock('POST', '/secrets', { secrets: 'fakeSecrets' });
+   * ```
+   *
+   * Multiple requests
+   *
+   * ```js
+   * I.mock('GET', ['/secrets', '/v2/secrets'], 403);
+   * ```
+   */
+  async mock(method, oneOrMoreUrls, dataOrStatusCode) {
```

Comment:
I just figured out that the name should be more informative. Like `mockRequest`

```js
I.mockRequest()
```
As there can probably be other things that can be mocked...

Reply:
Yah!, that name of the method will be more meaningful 😉